lib/Command/ACL.php

```diff
@@ -117,8 +117,16 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 			$mappingType = $input->getOption('user') ? 'user' : 'group';
 			$mappingId = $input->getOption('user') ?: $input->getOption('group');
 			$path = $input->getArgument('path');
+			if (!is_string($path)) {
+				$output->writeln('<error><path> argument has to be a string</error>');
+				return -3;
```

Comment:
`-3`?
Why?
Maybe `self::INVALID`? (would be `2`)

Reply:
I just copied this behavior from other checks above :woman_shrugging: